volatility3/framework/plugins/linux/mountinfo.py

```diff
@@ -247,6 +247,31 @@ def _generator(
                 "Could not filter by mount namespace id. This field is not available in this kernel."
             )
 
+    def get_superblocks(self):
+        """Yield file system superblocks based on the task's mounted filesystems.
+
+        Yields:
+            super_block: Kernel's struct super_block object
+        """
+        # No filter so that we get all the mount namespaces from all tasks
+        pid_filter = pslist.PsList.create_pid_filter()
+        tasks = pslist.PsList.list_tasks(
+            self.context, self.config["kernel"], filter_func=pid_filter
+        )
+
+        seen_sb_ptr = set()
+        for task, mnt, _mnt_ns_id in self._get_tasks_mountpoints(tasks):
```

Comment:
Thanks, good to know where to get the namespace if we later want to use it!  5:D